frameworks/JavaScript/nodejs/handlers/mongodb-raw.js

```diff
@@ -0,0 +1,84 @@
+var h = require('../helper');
+var async = require('async')
+var MongoClient = require('mongodb').MongoClient;
+var collections = {
+  World: null,
+  Fortune: null
+};
+MongoClient.connect('mongodb://127.0.0.1/hello_world?maxPoolSize=5', function (err, db) {
+  if (err) { throw err; }
+  collections.World = db.collection('world');
+  collections.Fortune = db.collection('fortune');
+});
+
+function mongodbRandomWorld(callback) {
+  collections.World.findOne({
+    id: h.randomTfbNumber()
+  }, function (err, world) {
+    world._id = undefined; // remove _id from query response
+    callback(err, world);
+  });
+}
+
+function mongodbGetAllFortunes(callback) {
+  collections.Fortune.find().toArray(function (err, fortunes) {
+    callback(err, fortunes);
+  })
+}
+
+function mongodbDriverUpdateQuery(callback) {
+  collections.World.findAndModify({
+    id: h.randomTfbNumber()
+  }, [['_id','asc']], {
+    $set: {randomNumber: h.randomTfbNumber()}
+  }, {}, function(err, world) {
+    world.value._id = undefined; // remove _id from query response
+    callback(err, world.value);
+  });
+}
+
+module.exports = {
+
+  SingleQuery: function (req, res) {
+    mongodbRandomWorld(function (err, result) {
+      if (err) { throw err; }
```

Comment:
the callback is called asynchronously, throwing an error in such callback will not get caught anywhere and it might kill the app instance and force restart in cluster.
If you want to have asynchronous error handling, you need to switch to promises (I recommend bluebird for that) or generators (node12 have them)

But in this case you don't need to do anything other than just returning a http500 if `err` is truthy